dll/win32/crypt32/store.c

```diff
@@ -1269,7 +1269,7 @@ static LONG CRYPT_OpenParentStore(DWORD dwFlags,
     void *pvSystemStoreLocationPara, HKEY *key)
 {
     HKEY root;
-    LPCWSTR base;
+    static LPCWSTR base;
```

Comment:
This chance seems weird, there is no use of base without it being assigned here?